src/webgl/p5.RendererGL.Immediate.js

```diff
@@ -326,4 +335,29 @@ p5.RendererGL.prototype._drawStrokeImmediateMode = function() {
   this._pInst._pixelsDirty = true;
 };
 
+p5.RendererGL.prototype.initPointVertexBufferImmediate = function() {
```

Comment:
This function name feels a little bit long and misleading. It is doing more than simple initialization. One could argue that the line 51 with the call to `createBuffer()` is the actual immediate mode point vertex buffer initialization. This function is doing something a little closer to drawing. Could you change the name to something a little bit shorter and with an underscore to indicate its privacy like `_drawPoints(points, amt)`? Notice I also put arguments in that example. I think it would be fairly simple to collapse the immediate and retained mode versions of this into a single function at the renderer level.